cmd/nerdctl/compose_build_linux_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestComposeBuild(t *testing.T) {
+	const imageSvc0 = "composebuild_svc0"
+	const imageSvc1 = "composebuild_svc1"
+
+	dockerComposeYAML := fmt.Sprintf(`
+services:
+  svc0:
+    build: .
+    image: %s
+    platform: amd64
+    ports:
+    - 8080:80
+  svc1:
+    build: .
+    image: %s
+    platform: arm64
+    ports:
+    - 8081:80
+`, imageSvc0, imageSvc1)
+
+	dockerfile := fmt.Sprintf(`FROM %s`, testutil.AlpineImage)
+
+	testutil.DockerIncompatible(t)
```

Comment:
Why incompatible?

Reply:
I saw a similar test [here](https://github.com/containerd/nerdctl/blob/2f70e700d5352c1d326c653e7659c017e9aaaca4/cmd/nerdctl/multi_platform_linux_test.go#L84). Seems not the case, I just removed it. Thanks.